pkg/restapi/issuer/operation/operations.go

```diff
@@ -273,6 +275,7 @@ func (c *Operation) prepareCreateCredentialRequest(data []byte, info *token.Intr
 	delete(subject, "updated_at")
 
 	req := &createCredential{
+		Context: []string{credentialContext},
```

Comment:
We will need to address adding different schemas for different requests (e.g. student card vs travel card). This can be handled later on.